custom_model_runner/datarobot_drum/drum/language_predictors/base_language_predictor.py

```diff
@@ -57,8 +58,11 @@ def configure(self, params):
             )
 
         model_metadata = read_model_metadata_yaml(self._code_dir)
-        self._schema_validator = (
-            SchemaValidator(model_metadata.get("typeSchema", {})) if model_metadata else None
+
+        # Set the environment variable STRICT_VALIDATION to use validation
+        self._schema_validator = SchemaValidator.create_validator(
+            model_metadata=model_metadata,
+            strict_validation=os.getenv("STRICT_VALIDATION") is not None,
```

Comment:
should use `os.getenv("STRICT_VALIDATION", False)` because this will be set to true even if `STRICT_VALIDATION=0`